charts/prometheus-smartctl-exporter/templates/daemonset.yaml

```diff
@@ -35,12 +35,24 @@ spec:
         {{- include "prometheus-smartctl-exporter.selectorLabels" $global | nindent 8 }}
         idx: i{{ $idx }}
     spec:
+      {{- if $global.Values.image.pullSecrets }}
+      imagePullSecrets:
+        {{- range $secret := $global.Values.image.pullSecrets }}
+        - name: {{ $secret }}
+        {{- end }}
+      {{- end }}
       containers:
       - image: "{{ $global.Values.image.repository }}:{{ $global.Values.image.tag | default $global.Chart.AppVersion }}"
         imagePullPolicy: {{ $global.Values.image.pullPolicy }}
         args:
         - '--smartctl.path={{ $config.smartctl_location }}'
         - '--smartctl.interval={{ $config.collect_not_more_than_period }}'
+{{- if $config.device_exclude }}
+        - '--smartctl.device-exclude={{ $config.device_exclude }}'
```

Comment:
The new fields should be entered in `values.yaml` with their default values.

Reply:
For the image pull secrets above I added an explicit default but I noticed here that the `config.devices` option does not have a default, so I followed the existing pattern in `values.yaml` and added commented out examples.  LMK if this is sufficient.

Reply:
Thank you, @jmif. Please, sign your commit.